### PR TITLE
Checksum in file status

### PR DESCRIPTION
**Pull request by flurdy**
Files uploaded will now include a SHA 256 hash in the file details

### CODE REVIEW

**Review by feedmypixel — APPROVED** [review]
LGTM 👍 

I suspect some fixtures and tests need the new `checksumSha256` property and value adding

File: src/server/common/helpers/scan-result-response.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ function toFilesResponse(uploadId, files) {
     fileStatus: file.fileStatus,
     contentType: file.contentType,
     contentLength: file.contentLength,
+    checksumSha256: file.checksumSha256,
```
Comment (christopherjturner) [inline src/server/common/helpers/scan-result-response.js:24]: maybe go for shorter name like just `sha256`?  The checksum prefix feels superfluous

Comment (flurdy) [reply src/server/common/helpers/scan-result-response.js:24]: Yeah maybe, but also for some sha256 might not mean much?

Comment (flurdy) [reply src/server/common/helpers/scan-result-response.js:24]: The other was just checksum, but then they would have to read the API to see if MD5, SHA1, SHA256, CRC32 etc. But it is at least shorter

Comment (flurdy) [reply src/server/common/helpers/scan-result-response.js:24]: After all we do not specify (by name) if the contentLength is bytes, Kb or Mb either.

Comment (flurdy) [reply src/server/common/helpers/scan-result-response.js:24]: :shrug:

Comment (flurdy) [reply src/server/common/helpers/scan-result-response.js:24]: Settled on keeping it as `checksumSha256`

**Pull request merged**